src/Plugins/ComplexCore/test/ConditionalSetValueTest.cpp

```diff
@@ -133,6 +135,17 @@ TEST_CASE("ConditionalSetValue: Test Algorithm Bool", "[ConditionalSetValue]")
   auto executeResult = filter.execute(dataGraph, args);
   REQUIRE(executeResult.result.valid());
 
+#if 0
+  // Write out the DataStructure for later viewing/debugging
+  std::string filePath = fmt::format("{}/ConditionalSetValueTest.dream3d", unit_test::k_BinaryDir);
+  // std::cout << "Writing file to: " << filePath << std::endl;
+  Result<H5::FileWriter> result = H5::FileWriter::CreateFile(filePath);
+  H5::FileWriter fileWriter = std::move(result.value());
+
+  herr_t err = dataGraph.writeHdf5(fileWriter);
+  REQUIRE(err >= 0);
+#endif
+
```

Comment:
```suggestion
```

Reply:
I'm going to keep this in here because writing a varied selection of DataStructure to HDF5 files is helping to find errors.